db/write_batch.cc

```diff
@@ -1609,6 +1611,8 @@ class MemTableInserter : public WriteBatch::Handler {
       // we are now iterating through a prepared section
       rebuilding_trx_ = new WriteBatch();
       rebuilding_trx_seq_ = sequence_;
+      // We only call MarkBeginPrepare once per batch, and the batch is
```

Comment:
nit: "the batch is initialized to false" => "unprepared_batch_ is initialized to false"